test/functional/feature_rbf.py

```diff
@@ -621,5 +637,64 @@ def test_replacement_relay_fee(self):
         tx.vout[0].nValue -= 1
         assert_raises_rpc_error(-26, "insufficient fee", self.nodes[0].sendrawtransaction, tx.serialize().hex())
 
+    def test_prechecks_overestimates_replacements(self):
```

Comment:
What's documented here is not such an important issue as the CVE in `test_no_inherited_signaling` but documenting an issue/limitation seems to have a precedent. Maybe this could make sense as an additional reference for the docs?